app/components/PortableTextStyled.tsx

```diff
@@ -30,19 +30,26 @@ export default function PortableTextStyled({ content }: { content: any }) {
 				h4: (props: any) => (
 					<h4 className="text-2xl xs:text-3xl mb-6 mt-12" {...props} />
 				),
-				normal: (props: any) => (
-					<p className="text-base mb-10" {...props} />
-				),
+				normal: (props: any) => <p className="text-base mb-10" {...props} />,
 				table: ({ rows }: { rows: Row[] }) => {
 					return (
 						<>
 							<table className={"w-full my-4"}>
 								<tbody>
-									{rows.map((row) => {
+									{rows.map((row, rowIndex) => {
```

Comment:
You should use the `<thead>`-tag for the header, should be placed above `<tbody>`